queries/runescript/highlights.scm

```diff
@@ -0,0 +1,117 @@
+; Variables
+(identifier) @variable
+
+(local_variable
+  "$" @attribute.builtin
+  name: (identifier) @variable)
+
+(game_variable
+  [
+    "%"
+    ".%"
+  ] @attribute.builtin
+  name: (identifier) @variable)
+
+(constant_variable
+  "^" @attribute.builtin
+  name: (identifier) @variable)
```

Comment:
Nit: all these `(identifier) @variable` highlights are unnecessary because of the first query

Reply:
Could you please elaborate further? I'm not sure I understand

Reply:
Ah, yes. Makes perfect sense. Thank you :)